src/Listing-pages/explore.css

```diff
@@ -0,0 +1,75 @@
+
+.badge-dot-to-separate{
+    margin: 0 0.2rem;
+}
+.video-info{
+    display: flex;
+}
+
+.avatar{
+    margin: 0.625rem 0 1rem;
+}
+
+.padding-right-4px{
+    padding-right: 0.25rem;
+}
+
+.category-list-item-bg{
+    height: 6.8rem;
+    background-color: var(--primary-color);
+}
+.category-link-active{
+    background-color: var(--primary-color);
+    color: white;
+}
+
+.category-list{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
could be simplified as: `grid-template-columns: repeat(4, 1fr);`

Reply:
Yeah, i will use this.